legend-engine-language-pure-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/from/data/embedded/ModelStoreDataParseTreeWalker.java

```diff
@@ -65,9 +71,30 @@ public ModelStoreData visit(ModelStoreDataParserGrammar.DefinitionContext ctx)
         for (ModelStoreDataParserGrammar.TypeIndexedInstancesContext typeIndexedInstancesContext : ctx.typeIndexedInstances())
         {
             String fullPath = PureGrammarParserUtility.fromQualifiedName(typeIndexedInstancesContext.qualifiedName().packagePath() == null ? Collections.emptyList() : typeIndexedInstancesContext.qualifiedName().packagePath().identifier(), typeIndexedInstancesContext.qualifiedName().identifier());
-
-            ValueSpecification instances = collection(typeIndexedInstancesContext.instance().stream().map(this::visitInstance).collect(Collectors.toList()));
-
+            ValueSpecification instances = null;
+            if (typeIndexedInstancesContext.instance() != null)
+            {
+                instances = collection(typeIndexedInstancesContext.instance().stream().map(this::visitInstance).collect(Collectors.toList()));
+            }
+            if (typeIndexedInstancesContext.embeddedData() != null)
+            {
+                EmbeddedData embeddedData = HelperEmbeddedDataGrammarParser.parseEmbeddedData(typeIndexedInstancesContext.embeddedData(), this.walkerSourceInformation, extensions);
+                if (embeddedData instanceof DataElementReference)
+                {
+                    PackageableElementPtr ptr = new PackageableElementPtr();
+                    ptr.fullPath = ((DataElementReference) embeddedData).dataElement;
+                    PackageableElementPtr bindingPtr = new PackageableElementPtr();
+                    bindingPtr.fullPath = "generated::default__generatedBindingForTestData";
```

Comment:
Do we have a test with 2 mappings using testable, would having fixed package conflict?